examples/source/interactivity-dynamic-content/Restrict_Duplicate_Selection.html

```diff
@@ -0,0 +1,92 @@
+<!---
+author: elisameyer
+formats:
+  - websites
+tags:
+  - dynamic-content
+--->
+
+<!-- ## Introduction -->
+<!--
+This is a sample showing how to restrict duplicate selection of items in amp-list. This uses
+'amp-selector' in combination with 'amp-bind' to check whether or not an item is currently in
```

Comment:
You can use ` to code highlight comments, e.g. `amp-list`:

```
This is a sample showing how to restrict duplicate selection of items in `amp-list`....
```

It'd be great if you could do this every time you mention an AMP component or a piece of code (e.g. variable names).